<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Nagendra-Yoga" ignored="0" name="Validating User login datadriven" tests="9" failures="3" timestamp="13 Aug 2021 22:59:46 GMT" time="424.024" errors="0">
  <testcase name="TC_loginDDT" time="29.388" classname="com.DollarDays.testCases.LoginDataDriverTest"/>
  <testcase name="TC_loginDDT" time="33.37" classname="com.DollarDays.testCases.LoginDataDriverTest">
    <failure type="java.lang.AssertionError" message="Could not sign in to DollarDays with valid user user123@gmail.com and valid password pwd2. TestCase Failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Could not sign in to DollarDays with valid user user123@gmail.com and valid password pwd2. TestCase Failed. expected [true] but found [false]
at com.DollarDays.testCases.LoginDataDriverTest.TC_loginDDT(LoginDataDriverTest.java:80)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- TC_loginDDT -->
  <testcase name="TC_loginDDT" time="2.955" classname="com.DollarDays.testCases.LoginDataDriverTest"/>
  <testcase name="TC_loginDDT" time="1.924" classname="com.DollarDays.testCases.LoginDataDriverTest"/>
  <testcase name="TC_loginDDT" time="2.374" classname="com.DollarDays.testCases.LoginDataDriverTest"/>
  <testcase name="TC_loginDDT" time="0.835" classname="com.DollarDays.testCases.LoginDataDriverTest"/>
  <testcase name="TC_loginDDT" time="0.508" classname="com.DollarDays.testCases.LoginDataDriverTest"/>
  <testcase name="TC_loginDDT" time="10.032" classname="com.DollarDays.testCases.LoginDataDriverTest">
    <failure type="java.lang.AssertionError" message="Signed  in to DollarDays successfully with invalid user srikanthtesting100@gmail.com and invalid password password123. Testcase failed. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Signed  in to DollarDays successfully with invalid user srikanthtesting100@gmail.com and invalid password password123. Testcase failed. expected [false] but found [true]
at com.DollarDays.testCases.LoginDataDriverTest.TC_loginDDT(LoginDataDriverTest.java:90)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- TC_loginDDT -->
  <testcase name="TC_loginDDT" time="305.787" classname="com.DollarDays.testCases.LoginDataDriverTest">
    <failure type="java.lang.AssertionError" message="Could not sign in to DollarDays with valid user team6testing@gmail.com and valid password password123. TestCase Failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Could not sign in to DollarDays with valid user team6testing@gmail.com and valid password password123. TestCase Failed. expected [true] but found [false]
at com.DollarDays.testCases.LoginDataDriverTest.TC_loginDDT(LoginDataDriverTest.java:71)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- TC_loginDDT -->
</testsuite> <!-- Validating User login datadriven -->
